docs/ots-sanitize.1

```diff
@@ -0,0 +1,51 @@
+.TH OTS-SANITIZE "1" "November 2020" "OpenType Sanitizer" "User Commands"
+.SH NAME
+ots-sanitize \- font validator and transcoder
+.SH SYNOPSIS
+.B yes
+[\fI\,OPTIONS\/\fR]... \fI\,FONT_FILE\/\fR [\fI\,DEST_FONT_FILE\/\fR] [\fI\,FONT_INDEX\/\fR]
+.SH DESCRIPTION
+.\" Add any additional description here
+.PP
+ots-sanitize is a program which validates and/or transcodes a font file using
+the OTS library:
+.PP
+.RS
+.nf
+transcoded_font = ValidateAndTranscode(original_font);
+if (validation_error)
+  PrintErrorAndExit;
+OutputToStdout(transcoded_font);
```

Comment:
Not sure why this pseudo code is here, without even a mention what it is, but I find it strange for man page.

Reply:
This was copy-pasted from `DesignDoc.md`, removed in new patchset.